lib/dstu2_resources.rb

```diff
@@ -744,6 +756,17 @@ module Resources
               ],
               "text": "Pyogenic Arthritis Involving Multiple Sites"
             },
+            "category": {
+              "coding": [
+                {
+                  "system": "http://argonaut.hl7.org/extension-codes",
```

Comment:
After offline conversation, it was decided the system is correct and this would not be a defect.